samples/Web/Controllers/TodosController.fs

```diff
@@ -6,6 +6,7 @@ open TodoBackend
 
 type FromClientIdHeaderAttribute() = inherit FromHeaderAttribute(Name="COMPLETELY_INSECURE_CLIENT_ID")
 
+[<CLIMutable>]
```

Comment:
`CLIMutable` attribute is necessary else we hit the following exception:

```
System.NotSupportedException: Deserialization of reference types without parameterless constructor is not supported. Type 'TodoBackend.Controllers.TodoView'
```

Reply:
I believe this is best remediated via `AddNewtonsoftJson` ? https://github.com/jet/dotnet-templates/blob/master/equinox-web-csharp/Web/Startup.cs#L39

Reply:
Yeah that seems to work, I had to add `Microsoft.AspNetCore.Mvc.NewtonsoftJson`.

Reply:
cool, thanks